WcaOnRails/config/locales/en.yml

```diff
@@ -65,20 +97,28 @@ en:
     labels:
       defaults:
         event_ids: "Events"
+        guests: "Guests"
+        comments: "Comments"
       competition:
         id: "ID"
     hints:
       defaults:
         dob: "Enter your date of birth in the format YYYY-MM-DD."
       user:
         name_html: "Enter your full name correctly, for example <strong>Stefan Pochmann</strong>. Not sloppily like <strong>s pochman</strong>."
+        email: "Changing your email will require confirming the new email."
+        pending_avatar: "After uploading a new avatar, you will have to wait for the Board to approve it."
       competition:
         competition_id_to_clone: "Optional. Can save you some typing if you're running a similar competition to a past competition."
         name: "The full name of the competition including the year at the end (e.g., European Rubik's Cube Championship 2006). Be sure to capitalize."
         cellName: "A short name for displaying. If the name of the competition is already short, you can re-use the name (e.g., Europe 2006)."
         cityName: "Name of the city and state (if applicable) where the competition takes place. Do not include country (e.g., Paris OR San Francisco, California)."
-        venueDetails: "Details about the venue (e.g., On the first floor far in the back, follow the signs)."
```

Comment:
This entry has moved [here](https://github.com/cubing/worldcubeassociation.org/pull/460/files#diff-3b35e11b9e261cae013c8cd8c46ecae1R302) because we need to inject the markdown message, and therefore we can't rely anymore on `simple_form`'s default behavior for looking up hints.